jmix-gradle-plugin/src/main/groovy/io/jmix/gradle/JmixPlugin.groovy

```diff
@@ -152,4 +155,71 @@ class JmixPlugin implements Plugin<Project> {
         }
         return result ?: 'unspecified'
     }
+
+    private static void registerCleanConfTask(Project project) {
+        project.tasks.register('cleanConf', Delete) {
+            doFirst {
+                def resources = project.file("src/main/resources/")
+                if (resources.exists() && resources.isDirectory()) {
+                    def mainProperties = new Properties()
+                    project.file("src/main/resources/application.properties").withInputStream { mainProperties.load(it) }
+
+                    def confDir = resolveConfDir(project, mainProperties)
+
+                    println("Delete directory = ${confDir}")
```

Comment:
Use `org.gradle.api.logging.Logger#lifecycle` method. Remove "=" from the message.

Reply:
Done.